phpunit.xml.dist

```diff
@@ -1,63 +1,64 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
-    bootstrap="system/Test/bootstrap.php"
-    backupGlobals="false"
-    beStrictAboutOutputDuringTests="true"
-    colors="true"
-    columns="max"
-    failOnRisky="true"
-    failOnWarning="true"
-    cacheDirectory="build/.phpunit.cache">
-    <coverage
-        includeUncoveredFiles="true"
-        pathCoverage="false"
-        ignoreDeprecatedCodeUnits="true"
-        disableCodeCoverageIgnore="true">
-        <report>
-            <clover outputFile="build/logs/clover.xml"/>
-            <html outputDirectory="build/coverage/html" highLowerBound="80"/>
-            <text outputFile="build/coverage/coverage.txt"/>
-        </report>
-    </coverage>
-    <extensions>
-        <bootstrap class="Nexus\PHPUnit\Tachycardia\TachycardiaExtension">
-            <parameter name="time-limit" value="0.50" />
-            <parameter name="report-count" value="30" />
-        </bootstrap>
-    </extensions>
-    <testsuites>
-        <testsuite name="System">
-            <directory>tests/system</directory>
-        </testsuite>
-    </testsuites>
-    <source>
-        <include>
-            <directory suffix=".php">system</directory>
-        </include>
-        <exclude>
-            <directory>system/Commands/Generators/Views</directory>
-            <directory>system/Debug/Toolbar/Views</directory>
-            <directory>system/Pager/Views</directory>
-            <directory>system/ThirdParty</directory>
-            <directory>system/Validation/Views</directory>
-            <file>system/bootstrap.php</file>
-            <file>system/ComposerScripts.php</file>
-            <file>system/Config/Routes.php</file>
-            <file>system/Test/bootstrap.php</file>
-            <file>system/Test/ControllerTester.php</file>
-            <file>system/Test/FeatureTestCase.php</file>
-        </exclude>
-    </source>
-    <php>
-        <server name="app.baseURL" value="http://example.com/"/>
-        <server name="CODEIGNITER_SCREAM_DEPRECATIONS" value="1"/>
-        <!-- Directory containing phpunit.xml -->
-        <const name="HOMEPATH" value="./"/>
-        <!-- Directory containing the Paths config file -->
-        <const name="CONFIGPATH" value="./app/Config/"/>
-        <!-- Directory containing the front controller (index.php) -->
-        <const name="PUBLICPATH" value="./public/"/>
-    </php>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
what changed here? line indents?

Reply:
Yes. Enable showing deprecations in console - autoreformat file.